scanomatic/io/pickler.py

```diff
@@ -45,7 +45,7 @@ def __init__(self):
         self._next = None
         self.io = None
 
-    def __call__(self, line: str):
+    def __call__(self, line: bytes):
```

Comment:
This file makes me nervous by proxy! Is it still useful, even after moving to json configs?

Reply:
unfortunately this is used for saving things that are not `Model`s... so yes this is a very hackish thing that should be removed some day hopefully.

Reply:
Good thing you have a decent store of bubbly beverages for all these upcoming moments of celebration.